bin/rules.d/openmoteb.rules

```diff
@@ -0,0 +1,2 @@
+# open nodes openmoteb
+KERNEL=="ttyUSB1" , SUBSYSTEMS=="usb", ATTRS{idVendor}=="0403" , ATTRS{idProduct}=="6010" , MODE="0664", GROUP="dialout", SYMLINK+="iotlab/ttyON_OPENMOTEB"
```

Comment:
I think this is wrong and is copy-pasted from M3/Fox rules. How can this work ? Did you change the IDs in the programmer ?

Reply:
> Some comments:
> 
> * it's possible to use an elf, just convert it to bin before flashing. We did similar things in the past for edbg (converting from elf to hex)

Yes indeed, we can to that. So we will assume to flash the bin files at a default offset.

> * there are a lot of style/lint issues reported by the CI

As usual :-)

> * the segger programmer class has no unit test. I know they are dumb but are useful to check things behave as they should and ensure important lines are covered by tests

Ok need to improve this too.

> * I haven't checked in details but is the segger programmer integrated in the `iotlab_flash` cli ? That would be useful of an openmote is plugged behind an open Linux

No, this could be another improvement in another PR.

> * There's something strange with the udev rules, idProduct/idVendor fields are the same as for the M3/Fox but it's a different thing (a JLink Mini Edu or something I guess). I think it would be useful to document the setup somewhere, maybe in the opennode docstring.

Will check that.

Thanks for the comments.